es-css-modules/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>es-css-modules - CSS in JS</title>
+    <link rel="stylesheet" href="bundle.css">
+  </head>
+  <body>
+    <div id="content"></div>
+    <script src="bundle.js"></script>
```

Comment:
`bundle.js` doesn't exists, it should be created (with `babel` or `webpack`) during the build process.